IonDotnet/Tree/Impl/IonValue.cs

```diff
@@ -8,13 +8,13 @@
 using IonDotnet.Systems;
 using IonDotnet.Utils;
 
-namespace IonDotnet.Tree
+namespace IonDotnet.Tree.Impl
 {
     /// <summary>
     /// Represents a tree view into Ion data. Each <see cref="IonValue" /> is a node in the tree. These values are
     /// mutable and strictly hierarchical. 
     /// </summary>
-    public abstract class IonValue
+    public abstract class IonValue : IIonValue

```

Comment:
This should now be internal

Reply:
Done. Unit test access modifiers had to be changed too, to reflect the change in this modifier from public to protected.